lms/data_tasks/report/create_from_scratch/04_activity_counts/01_groups/04_group_type_counts/01_create_view.jinja2.sql

```diff
@@ -0,0 +1,16 @@
+DROP MATERIALIZED VIEW IF EXISTS report.group_type_counts CASCADE;
+
+-- A weekly count of group annotation types limited to the correct authority
+
+CREATE MATERIALIZED VIEW report.group_type_counts AS (
+    SELECT
+        created_week,
+        group_id,
+        sub_type,
+        shared,
+        count
+    FROM h.annotation_type_group_counts
+    JOIN h.authorities ON
+        annotation_type_group_counts.authority_id = authorities.id
+        AND authorities.authority = '{{ region.authority }}'
+) WITH NO DATA;
```

Comment:
We pull the specific authority through here (e.g. Canada will only get Canadian LMS stuff here). There's not much interpretation in this case.